qa/workunits/windows/libvirt_vm/setup.sh

```diff
@@ -77,7 +77,7 @@ sudo virt-install \
     --network network=default,model=virtio \
     --controller type=virtio-serial \
     --channel unix,target_type=virtio,name=org.qemu.guest_agent.0 \
-    --noautoconsol
+    --noautoconsole
```

Comment:
notable

Reply:
Yup -- libvirt must be accepting contracted options.